notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettOppgaveLoeser.kt

```diff
@@ -34,6 +37,10 @@ class OpprettOppgaveLoeser(
             behov[DataFelt.ORGNRUNDERENHET].asText(),
             behov[DataFelt.VIRKSOMHET].asText()
         )
+        if (oppgaveId.isBlank()) {
+            publishFail(behov.createFail("OpprettOppgave feilet"))
+            return
+        }
```

Comment:
Foretrekker if-else over `return` midt i koden. Da blir det lettere å følge flyten.

Reply:
Dette var for å være konsistent med linjene over der det samme gjøres for forespørselId. 
Er ikke heeelt enig i at det nødvendigvis er enklere å lese masse if / else

Reply:
Det var gjemt i diffen. Da er funksjonen allerede tainted, så 🤷 

Jeg er kanskje hardliner, men synes det er lettere å følge flyten når det kun finnes én return, så slipper jeg å lete i koden etter alle.